docs/ru/easytofly.md

```diff
@@ -0,0 +1,142 @@
+## Информация о команде
+Название команды: EasyToFly
```

Comment:
Please, provide e-mail, Telegram, of something for contact.

Reply:
https://github.com/CopterExpress/clover/pull/309/commits/57373f3b3503b7c29e26c54b4fa3ecc237e76fe2